### ISSUE TITLE
article

### ISSUE THREAD

**Issue by LangInteger** [LangInteger/learning#2]
There is no shortage of opinions on the best way to stay in shape. In my opinion, I believe a combination of well-balanced diet and exercise routine is the best way to stay healthy and there has already formed a trend in society. I feel this way for two main reasons, which I will explore in the following paragraphs.

The reasons for this trend may involve the recognition people who do not regulate their diet and exercise time by time are weak in terms of both physical and mental health. They are more likely to be vulnerable to season flues and show laziness in daily work. By contrast, those who have taken diet and exercise more seriously, will be stronger in front of diseases and illness. They tend to be full of energy, which is very important for their superior performance in study and work, as well as giving them courage in terms of solving problems in their lives.

Advocates of this trend believe that today’s fast-paced lifestyle and stressful working conditions mean that people often ignore the condition of their bodies. Even if we have developed good relationship with colleagues and other people in daily life, we may just share unhealthy habits together and has nothing to do with improving the quality of life. If there were people with good balanced diet and enough exercise around you, you would be more likely to learn from them and take your life better, rather than maintain an unhealthy life cycle and never treat it as a problem.

In conclusion, I do believe that good health is best achieved by treating the body to good food and moderate exercise. This is because these two habits will eventually lead to a reduction in weight and an increase in overall well-being.
